pipeline/api/cloud.py

```diff
@@ -287,17 +307,35 @@ def upload_pipeline(
 
         new_variables: List[PipelineVariableGet] = []
         print("Uploading variables")
+
+        from pipeline.objects import PipelineFile
```

Comment:
is there a reason we need to have these imports here?

Reply:
Unfortunately yes, cyclical dependencies.